spire/templates/apps/feeder.yml

```diff
@@ -983,6 +983,7 @@ Resources:
             Resource: !GetAtt FeederUploadsBucket.Arn
           - Action:
               - s3:AbortMultipartUpload
+              - s3:DeleteObject
```

Comment:
Would this also need `s3:DeleteObjectVersion`, for `s3.delete_object` calls?

Reply:
Yes, for a versioned bucket I think you need `DeleteObjectVersion` to do any deleting